sponsor.html

```diff
@@ -0,0 +1,316 @@
+      alt="Major League Hacking 2021 Hackathon Season" style="width:100%"></a>
+
+  <div class="contact-footer">
+    <div class="contact section text-center">
+      <div class="social-buttons">
+        <a href="https://www.youtube.com/channel/UC-Wo4UqhGUKQEp4o-51VMIw/" target="_blank">
+          <div class="purple-circle">
+            <img src="./img/w_youtube.svg" alt="YouTube icon">
+          </div>
+        </a>
+        <a href="https://twitter.com/gotechnica?lang=en" target="_blank">
+          <div class="purple-circle">
+            <img src="./img/w_twitter.svg" alt="Twitter icon">
+          </div>
+        </a>
+        <a href="https://www.facebook.com/gotechnica/" target="_blank">
+          <div class="purple-circle">
+            <img src="./img/w_facebook.svg" alt="Facebook icon">
+          </div>
+        </a>
+        <a href="https://www.instagram.com/gotechnica/" target="_blank">
+          <div class="purple-circle">
+            <img src="./img/w_instagram.svg" alt="Instagram icon">
+          </div>
+        </a>
+        <a href="https://medium.com/gotechnica" target="_blank" >
+          <div class="purple-circle">
+            <img src="./img/w_medium.svg" alt="Medium icon">
+          </div>
+        </a>
+        <a href="https://linkedin.com/company/gotechnica" target="_blank">
+          <div class="purple-circle">
+            <img src="./img/w_linkedin.svg" alt="LinkedIn icon">
+          </div>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="footer" style="background-color: #E9E3ED;">
+    <div class="container" style="padding: 1%;">
+      <div id="footer-links">
+        <!--<a href="https://gotechnica.org/sponsor" target="_blank">Sponsor</a> | --><a href="./files/terms.pdf" target="_blank">Code of Conduct</a> <!--| <a href="https://gotechnica.org/resources" target="_blank"> Resources</a> | <a href="https://gotechnica.org/donate" target="_blank"> Donate</a>-->
+      </div>
+      <span style="opacity: .7">Copyright © 2021 Technica</span>
+    </div>
+  </div>
+
+  <!-- mlh badge -->
+  <a id="mlh-trust-badge" style="display:block;max-width:100px;min-width:60px;position:fixed;right:50px;top:0;width:10%;z-index:10000" href="https://mlh.io/seasons/na-2021/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2021-season&utm_content=white" target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2021/mlh-trust-badge-2021-white.svg" alt="Major League Hacking 2021 Hackathon Season" style="width:100%"></a>
+</body>
+</html>
+
+<!-- Pie Chart -->
+<script>
+  var ctx = document.getElementById('edulevels');
+  colors = [
+    'rgb(0, 255, 255)',
+    'rgb(255, 123, 172)',
+    'rgb(45, 45, 45)',
+    'rgb(182, 161, 196)'
+  ]
+
+  var eduChart = new Chart(ctx, {
+    type: 'doughnut',
+    data: {
+      // labels: ['Undergraduate College', 'High School', 'Graduate College', 'Middle School'],
```

Comment:
"undergraduate/graduate college" seems redundant to me, you can just delete "college" from both